server/routes/bookedtoday/index.ts

```diff
@@ -1,4 +1,4 @@
-import express, { type Router } from 'express'
+import express, { Router } from 'express'
```

Comment:
Removed type as caused typecheck to fail in multiple places. Not sure if this is the correct fix.

Reply:
Don't think this is right - it doesn't fail for me. Did you definitely run npm install?

Support for adding `type` to individually named imports in the [latest version of typescript](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-5.html#type-modifiers-on-import-names)

Reply:
done